File: die.py
# implements the Die class for the 10000 dice game

from random import randint

class Die():
    def __init__(self):
        self.value = 1
        
    def roll(self):
        new_value = randint(1, 6)
        self.value = new_value
        print(self.value, end = " ")
            
    def get_value(self):
        return self.value
        

File: java/Die.java
import java.util.*;
import java.io.*;

public class Die {
    
    int value;
    Random random;
    
    public Die() {
        this.value = 1;
        random = new Random();
        
    }
    
    public void roll() { // probably change return type
        int new_value = random.nextInt(6) + 1;
        this.value = new_value;
        System.out.print(this.value + " ");
    }
    
    public int get_value() {
        return this.value;
    }
    
    public static void main(String[] args) {
        Die d = new Die();
        for (int i = 0; i < 6; i++) {
            d.roll();
        }
    }
}
